apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/edit/page.tsx

```diff
@@ -17,7 +17,8 @@ export default async function SurveysEditPage({ params }) {
     getActionClasses(params.environmentId),
     getAttributeClasses(params.environmentId),
   ]);
-  const isEncryptionKeySet = !!FORMBRICKS_ENCRYPTION_KEY;
+
+  const isEncryptionKeySet = !!ENCRYPTION_KEY;
```

Comment:
I think we can skip this as it's not possible to not have the ENCRYPTION_KEY set. So please also remove the disabled state of the encryption toggle together with the tooltip :-)